edxval/transcript_utils.py

```diff
@@ -1,16 +1,20 @@
 """
 A module containing transcripts utils.
 """
+# pylint: disable=inconsistent-return-statements
```

Comment:
I only see one place where a missing `return None` is probably triggering this, can we avoid the module-wide warning disable?